source/user_guide.rst

```diff
@@ -0,0 +1,93 @@
+User Guide
+==========
+
+The git packaging workflow is currently an opt-in mechanism for submissions to
+openSUSE:Factory via `src.opensuse.org <https://src.opensuse.org>`_ instead of
+`OBS <https://build.opensuse.org/>`_. It allows packagers to maintain their
+package sources in the `pool <https://src.opensuse.org/pool>`_ organization
+instead of on OBS. Submissions are handled via pull requests on
+https://src.opensuse.org instead of submit requests on OBS.
+
+A bot has been setup that will automatically forward eligible pull requests as
+submitrequests directly to openSUSE:Factory.
+
+
+Packager Guide
+--------------
+
+.. hint::
+   Consider adjusting the package description to alert contributors that
+   ordinary submitrequests are no longer possible and where they should send a
+   pull request instead.
+
+To participate in the test drive, modify the development package in the
+developroject as follows:
+
+1. Login to https://src.opensuse.org using the openSUSE IDP provider.
+
+2. Clone your package from https://src.opensuse.org/pool/$pkg_name and switch to
+   the ``factory`` branch.
+
+3. Checkout your package from the configured develproject from OBS:
+
+.. code-block:: bash
+
+   osc co $(osc develproject openSUSE:Factory $pkgname)
+
+4. Compare your package in git (from step 2) with the package on OBS (from step
+   3). If there are differences, fork the package on src.opensuse.org and send a
+   pull request.
+
+5. Modify the package meta of the devel package via :command:`osc meta -e pkg
+   $develprj $pkgname` by adding the following entry, which will automatically
+   synchronize your package from git to OBS.
```

Comment:
This will also tell the openSUSE:Factory team that the package is maintained in git and submitrequests are expected no longer to come from the devel project but via the scm-staging bot that is mirroring pull requests.